inference-engine/tests/functional/inference_engine/ngraph_reader/binary_convolution_tests.cpp

```diff
@@ -0,0 +1,448 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadBinaryConvolutionNetwork) {
+    std::string model = R"V0G0N(
+<net name="model_bin" version="10">
+	<layers>
+		<layer id="0" name="612/placeholder_port_0" type="Parameter" version="opset1">
+			<data shape="1,64,56,56" element_type="f32"/>
+			<output>
+				<port id="0" precision="FP32">
+					<dim>1</dim>
+					<dim>64</dim>
+					<dim>56</dim>
+					<dim>56</dim>
+				</port>
+			</output>
+		</layer>
+		<layer id="1" name="612" type="MaxPool" version="opset1">
```

Comment:
Can we remove unnecessary operations from tests?

Reply:
Will investigate it today

Reply:
done